src/components/StatusChip.tsx

```diff
@@ -52,9 +52,10 @@ interface StatusChipProps {
   text: string;
   icon: 'check' | 'block' | 'hourglass';
   bgcolor?: string;
+  paddingY?: string | number;
```

Comment:
lurer på om vi heller skal tillate extending av sx enn å ta inn disse to eksplisitt

Reply:
æsj, sliter med å få riktige typer på det uten mye styr, så tror det er like greit å ta det sånn egentlig.